config/config.play.js

```diff
@@ -8,16 +9,13 @@ module.exports = {
   },
   dist: 'dist-play',
   postcss: config.postcss,
-  templateCompiler: true,
-  webpack: {
-    resolve: {
-      modules: [
-        path.resolve('src')
-      ]
-    }
+  webpack (config) {
+    ['app', 'preview'].forEach(entry => {
```

Comment:
Just add this in [poi@8.0.4](https://github.com/egoist/poi/releases/tag/poi%408.0.4)